atd-vze/src/Components/GridTableSearch.js

```diff
@@ -91,9 +88,12 @@ const GridTableSearch = ({
   return (
     <Col md={12}>
       <Form className="form-horizontal" onSubmit={handleSearchSubmission}>
-        {!isFieldSelected && searchFieldValue && (
+        {!isFieldSelected && isSearchValueEntered && (
           <Alert color="warning">Please provide a field to search.</Alert>
         )}
+        {isFieldSelected && !isSearchValueEntered && (
+          <Alert color="warning">Please enter a value to search.</Alert>
+        )}
```

Comment:
This was the most low-effort way to indicate that a search value is missing which is the opposite situation that already had a warning message to address. The warning messages can still only be cleared by clicking the reset button.

I think this seems okay but would like some input. 🙏